.github/ISSUE_TEMPLATE/01-compatible-certification-request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Compatible Certification Request
+about: Start a request for a compatible certification 
+title: 'MicroProfile Rest Client [Version] Compatible Certification Request'
```

Comment:
should be MicroProfile Context Propagation

Reply:
Thanks for spotting this. I corrected it in the update that also updates the asciidoctor version to match what mp rest client uses.